uaclient/actions.py

```diff
@@ -44,6 +46,11 @@
 
 USER_LOG_COLLECTED_LIMIT = 10
 
+# XXX handle /etc/apparmor.d/local/<name>, which has the local admin overrides
+APPARMOR_PROFILES = [
```

Comment:
This is just a nit, but maybe we should move this definition to the `defaults` module instead

Reply:
The main rationale is to just avoid potential circular dependencies in the future, as the `defaults` module doesn't import anything. Additionally, all of our settings are defined there, so if it is not a problem, I would vote to move it there.